bundles/core/src/test/java/com/adobe/cq/commerce/core/components/internal/models/v1/button/CategoryRetrieverTest.java

```diff
@@ -80,6 +74,7 @@ public void testCategoryUrlPathQueryFallback() {
 
     @Test
     public void testExtendCategoryQuery() {
+        // TODO: check if that is redundant with CategoryRetrieverTest#testCategoryUrlPathQueryById()
```

Comment:
Yes I think this is redundant, however with CIF-2062 & CIF-2153 the entire CategoryRetrieverTest.java for button component will not be needed. So ok for now.